src/riscv_core/veer_el2/config/compile.yml

```diff
@@ -1,76 +1,85 @@
 ---
-provides: [mcu_el2_veer_pkg]
+provides: [veer_el2_defines_pkg]
```

Comment:
Since MCU is renamed with the css_mcu0 prefix everywhere else, shall we also name the package as such here?